arrow-instances/src/main/kotlin/arrow/instances/tuple.kt

```diff
@@ -1,9 +1,52 @@
 package arrow.instances
 
-import arrow.typeclasses.Eq
-import arrow.typeclasses.Monoid
+import arrow.*
 import arrow.core.*
-import arrow.instance
+import arrow.core.Either.*
+import arrow.typeclasses.*
+
+@instance(Tuple2::class)
+interface Tuple2FunctorInstance<F> : Functor<Tuple2KindPartial<F>> {
```

Comment:
NIT: F is used for witness types, but I cannot suggest a better option :D

Reply:
In the case of applied types not in higher kinded position the convention we are following is `A`, `B`,... if higher kinded then `F`, `G`... In this case it should be `A` since the left is also that in `Tuple2<A, B>`